File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> s;
        int k=0;
        for(auto i : nums)
        {
            s.insert(i);
        }
        for(auto itr=s.begin();itr!=s.end();itr++)
        {
            nums[k++]=*itr;


        }
        return k;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        dic = defaultdict(int)

        for i in nums :
            dic[i]+=1
        j=0
        for i in dic.keys():
            nums[j]=i
            j+=1

        return len(dic.keys())



        